backends/dpdk/DpdkXfail.cmake

```diff
@@ -26,7 +26,7 @@ p4c_add_xfail_reason("dpdk"
   testdata/p4_16_samples/pna-dpdk-invalid-hdr-warnings5.p4
   testdata/p4_16_samples/pna-dpdk-invalid-hdr-warnings6.p4
   testdata/p4_16_samples/pna-dpdk-header-union-stack2.p4
-  testdata/p4_16_samples/dash/dash-pipeline-pna-dpdk.p4
+  testdata/p4_16_dpdk_errors/psa-dpdk-128bitCast_error.p4
```

Comment:
I thought that for tests in the p4_16_dpdk_errors (and other testdata/*_errors directories), a passing test is when the compiler produces the expected error messages in the output, which you have as part of this PR for psa-dpdk-128bitCast_error.p4, so that is good.  But why is it in the list of tests marked as expected to fail?  Do DPDK tests use Xfail differently than other p4c test programs?